### PR TITLE
Upgraded to Python 3

### PR DESCRIPTION
**Pull request by avishkar58**
Finally upgrading to Python 3! 
Code converted with 2to3, all requirements upgraded to latest versions possible and tested locally (only Werkzeug had an incompatibility and needed to remain on a previous version)

### CODE REVIEW

**Comment by amritpurshotam** [comment]
This is great, nice one! There's a lot to get through here so will have a look properly tomorrow

File: api/requirements.txt [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
-Flask-Admin==1.5.3
-Flask-Bcrypt==0.7.1
-Flask-Failsafe==0.2
-Flask-Migrate==2.3.0
-Flask-RESTful==0.3.7
-Flask-Redis==0.3.0
-flask-restplus==0.13.0
-Flask-SQLAlchemy==2.3.2
-Flask-Script==2.0.6
+Flask
```
Comment (amritpurshotam) [inline api/requirements.txt:1]: Should we completely unpin the package versions? With the upgrade I think the case can be made to take the latest versions now can also see how that would be quite tedious 🙈

**Review by amritpurshotam — APPROVED** [review]
Nicely done! Glad this is finally out the way 😄 And dare I say was this upgrade easy? It seemed a lot harder when it was tried last year

File: api/requirements.txt (hunk 1 above)
Comment (avishkar58) [reply api/requirements.txt:1]: They are completely unpinned now except for Werkzeug which has a version incompatibility with the latest Flask-restful, unless I didn't understand correctly what you mean?

**Comment by avishkar58** [comment]
Thanks for the review @amritpurshotam ! I know it was a large one! It was actually easier than I thought it would be, 2to3 did most of the heavy lifting and there was only a handful of changes I had to make manually mostly to do with bytes vs unicode incompatibilities :)

**Pull request merged**